src/labs.txt

```diff
@@ -3272,6 +3272,40 @@ receiving the push.  It is possible to set it up automatically, but I
 _never_ remember the commands to do that.  Check out the "Git Remote
 Branch" gem for easy management of remote branches.
 
+h2. You Can Push Anything
+
+p. Pushing to a remote repository requires specifying what you are pushing and
+where you are pushing it. The @git push shared main@ command uses a shorthand
+to specify where the branch is going to be pushed. You could read it like this:
+@git push shared main:main@, where what comes before the colon is in your local
+repository and what comes after is in the remote repository. There is a lot of
+power in this ability. Recall Lab 24 where we looked at the internals of the
```

Comment:
nit: Should we avoid referencing labs by number?  They could easily wind up renumbered as stuff moves around, right?

Reply:
@joealba I added some auto-linking for it so you can reference the lab by title in the text and it'll insert a link. It's in the preview now.